Builds/Test.py

```diff
@@ -306,7 +306,9 @@ def run_tests(args):
                 quiet = '-q'
             if ARGS.testjobs:
                 testjobs = ('--unittest-jobs=' + str(ARGS.testjobs))
-            resultcode, lines = shell(executable, (testflag, quiet, testjobs,))
+                resultcode, lines = shell(executable, (testflag, quiet, testjobs,))
+            else:
+                resultcode, lines = shell(executable, (testflag, quiet,))
```

Comment:
Woops. This is a good catch. However, instead of having a branch here, could you initialize `testjobs = ''` like we do with the other vars around line 302?

Reply:
Sure, just a sec.